parquet-plugins/parquet-encoding-vector/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.parquet</groupId>
     <artifactId>parquet</artifactId>
-    <version>1.15.0-SNAPSHOT</version>
+    <version>1.15.1-SNAPSHOT</version>
```

Comment:
Why is this changed? Is it because this was overlooked before?

Reply:
The release process does not automatically update this...

Reply:
OK, maybe we should fix it later.